utils/resource/no_usage.go

```diff
@@ -10,6 +10,10 @@ var NoUsage User = noUsage{}
 
 type noUsage struct{}
 
+func (noUsage) AvailableDiskPercentage() uint64 {
+	return math.MaxUint64
```

Comment:
does this need to be swapped?  `AvailableDiskBytes` = `math.MaxUint64` and `AvailableDiskPercentage` = 100?